tracer/src/Datadog.Trace/ClrProfiler/InstrumentationCategory.cs

```diff
@@ -2,14 +2,21 @@
 // Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
+#nullable enable
 
-namespace Datadog.Trace.ClrProfiler
+using System;
+
+namespace Datadog.Trace.ClrProfiler;
+
+[Flags]
+internal enum InstrumentationCategory : uint
 {
-    [System.Flags]
-    internal enum InstrumentationCategory
-    {
-        Tracing = 1,
-        AppSec = 2,
-        Iast = 4
-    }
+    // NOTE : When modifying this file make sure to update the InstrumentationCategory enum in the CodeGenerator folder inside the _build project
+
+    Tracing = 1,
+    AppSec = 2,
+    Iast = 4,
+    Rasp = 8,
```

Comment:
So far, I think that there are no Rasp aspects. Still, it does not harm adding this enum.